### PR TITLE
Created a model for exam

### LINKED ISSUE #210: [Feat]: [Create a model for exam]
## Description

Create an exam model

This model will be named exam.js

create a schema with key and value pairs with appropriate conditions.
Models should be added to the model folder.

## Proposed Solution
```
date: date
startTime: time
duration: int
supervisor: Faculty 
infrastructure: Infrastructure
course: Course
```

### PR DESCRIPTION
**Pull request by Abhishekgoyal007**
## Description
Created a model for exam 

## Fixes
#210 

## Checklist

- [x] Code follows project's style guidelines.
- [x] Changes are documented appropriately.

### CODE REVIEW

**Review by TejasNair9977 — APPROVED** [review]
LGTM

**Comment by Aastha-S-Rai** [comment]
Looking at other models, I think 'e' in exam should be Capital like 'Exam'. @TejasNair9977 ??

File: models/exam.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+const examSchema = {
+  date: { type: Date, required: true },
+  startTime: { type: Date, required: true },
+  duration: { type: Number, required: true },
+  supervisor: { type: connector.Schema.Types.ObjectId, ref: "Faculty", required: "true" },
+  infrastructure: { type: connector.Schema.Types.ObjectId, ref: "Infrastructure", required: "true" },
+  course: { type: connector.Schema.Types.ObjectId, ref: "Course", required: "true" },
+};
+
+const exam = connector.model("exam", examSchema);
```
Comment (TejasNair9977) [inline models/exam.js:12]:
```suggestion
const exam = connector.model("Exam", examSchema);
```

**Review by TejasNair9977 — APPROVED** [review]
LGTM

**Pull request merged**